provider/ec2/local_test.go

```diff
@@ -1161,6 +1161,37 @@ func (t *localServerSuite) testStartInstanceAvailZoneAllConstrained(c *gc.C, run
 	c.Assert(errors.Details(err), jc.Contains, runInstancesError.ErrorMessage())
 }
 
+// addTestingNetworkInterface adds one network interface with vpc id and
+// availability zone. It will also have a private IP with no
+func (t *localServerSuite) addTestingNetworkInterfaceToInstance(c *gc.C, instId instance.Id) ([]instance.Id, string) {
+	vpc := t.srv.ec2srv.AddVpc(types.Vpc{
+		CidrBlock: aws.String("0.1.0.0/16"),
+		IsDefault: aws.Bool(true),
+	})
+	sub, err := t.srv.ec2srv.AddSubnet(types.Subnet{
+		VpcId:            vpc.VpcId,
+		CidrBlock:        aws.String("0.1.2.0/24"),
```

Comment:
> 0.1.2.0

There is a suggestion to use valid IP, even in test examples.

Reply:
I'll change to 10.x.x.x where ever this occurs